packages/cli/src/lib/utils.ts

```diff
@@ -76,30 +76,12 @@ export const getAxeSource = (axePath?: string): string | void => {
     return;
   }
 
-  // Look for axe in current working directory
-  if (!axePath) {
-    axePath = path.join(process.cwd(), 'axe.js');
+  if (axePath && fs.existsSync(axePath)) {
+    return fs.readFileSync(axePath, 'utf-8');
   }
 
-  if (!fs.existsSync(axePath)) {
-    // Look for axe in CWD ./node_modules
-    axePath = path.join(process.cwd(), 'node_modules', 'axe-core', 'axe.js');
-  }
-
-  if (!fs.existsSync(axePath)) {
-    // `__dirname` is /@axe-core/cli/dist/src/lib when installed globally
-    // to access the locally installed axe-core package we need to go up 3 levels
-    // if all else fails, use the locally installed axe
-    axePath = path.join(
-      __dirname,
-      '..',
-      '..',
-      '..',
-      'node_modules',
-      'axe-core',
-      'axe.js'
-    );
-  }
+  const a = require.resolve('axe-core');
+  axePath = path.join(a, '..', 'axe.js');
```

Comment:
I have never understood why we didn't just do `require.resolve('axe-core')` here, but I'm 100% sure someone said there was a reason for it.